ledger/block/src/transaction/deployment/mod.rs

```diff
@@ -131,6 +162,16 @@ impl<N: Network> Deployment<N> {
         &self.program
     }
 
+    /// Returns the program checksum, if it was stored.
+    pub const fn program_checksum(&self) -> Option<&[U8<N>; 32]> {
+        self.program_checksum.as_ref()
+    }
+
+    /// Returns the program owner, if it was stored.
+    pub const fn program_owner(&self) -> Option<&Address<N>> {
+        self.program_owner.as_ref()
```

Comment:
```suggestion
    pub const fn program_owner(&self) -> Option<Address<N>> {
        self.program_owner
```
IIRC the `Address` is `Copy`

Reply:
the same might apply to the checksum

Reply:
I believe `console::U8` is copy.

Reply:
ah, I may have checked the circuit variant